crates/rpc/rpc-types/src/eth/block.rs

```diff
@@ -208,8 +208,6 @@ pub struct Header {
     pub mix_hash: H256,
     /// Nonce
     pub nonce: Option<H64>,
-    /// Size in bytes
-    pub size: Option<U256>,
```

Comment:
pretty crazy to me that `serde(flatten)` does not detect duplicate fields. is this a bug we should report upstream?